### ISSUE TITLE
ofSetBackgroundAuto(false) clears everything

### ISSUE THREAD

**Issue by AdaZhao1211** [openframeworks/openFrameworks#5831]
Hey guys,
I am new to OF and am trying some examples from [here](http://openframeworks.cc/ofBook/chapters/intro_to_graphics.html)

However, the ofSetBackgroundAuto(false) seems to clear everything on the screen. All I got was a black window.
I am using Windows 10, visual studio 2015, of 0.9.8

```
void ofApp::setup(){
	ofSetBackgroundAuto(false);
	ofBackground(0);
}

void ofApp::draw(){
	ofDrawRectangle(10, 10, 200, 200);
	if (ofGetMousePressed(OF_MOUSE_BUTTON_LEFT)) {
		ofSetColor(255);
		ofSetRectMode(OF_RECTMODE_CENTER);
		ofDrawRectangle(ofGetMouseX(), ofGetMouseY(), 50, 50);
	}
}
```

**Comment by hamoid** [comment]
Hi! The GitHub issues are for bugs and new features in OpenFrameworks, but there doesn't seem to be a bug here. Please post questions to http://forum.openframeworks.cc/

Your code seems to work fine. Click and drag draws rectangles on the screen without clearing the background between frames. And you start with a black window, which is expected.

**Comment by AdaZhao1211** [comment]
Hi hamoid, 
Thank you for the reply. I understand how to use the program but dragging my mouse didn't give anything. Without the `ofSetBackgroundAuto(false)`, the rectangle appears once and then gets cleared by the background. In my case, I can't even see that one-time blinking of the rectangle.
Also, since I added `ofDrawRectangle(10, 10, 200, 200);`, I assume that there should be a rectangle even if I don't do anything. However, I only got a black windows.

**Comment by hamoid** [comment]
I see! That is definitely not the expected behavior. Your example does work for me on Linux. 
Do any of the other issues related to `ofSetBackgroundAuto()` help? For instance https://github.com/openframeworks/openFrameworks/issues/2850 Do other OF examples work fine (2D and 3D graphics)?

What kind of graphics card do you have? Updating the drivers sometimes helps.

Another thing you could try is using the nightly builds from http://openframeworks.cc/download/ or even the version from GitHub, which has many fixes compared to 0.9.8.

**Comment by danzeeeman** [comment]
I cannot reproduce this bug.
